src/test/java/browserstack/stepdefs/BaseTest.java

```diff
@@ -0,0 +1,125 @@
+
+    	
+		 if (environment.equalsIgnoreCase("local")) {
+	            if (OsUtility.isMac()) {
+	                System.setProperty("webdriver.chrome.driver", chromeDriverBaseLocation+ "/chromeDriverMac/chromedriver");
+	            }
+	            if (OsUtility.isWindows()) {
+	                System.setProperty("webdriver.chrome.driver", chromeDriverBaseLocation+ "/chromeDriverWin/chromedriver.exe");
+	            }
+	            if (OsUtility.isUnix()) {
+	                System.setProperty("webdriver.chrome.driver",chromeDriverBaseLocation+ "/chromeDriverLinux/chromedriver");
+	            }
+	            ThreadLocalDriver.setWebDriver(new ChromeDriver());
+	        } else if (environment.equalsIgnoreCase("remote")) {
+	            JSONParser parser = new JSONParser();
+	            JSONObject config = (JSONObject) parser.parse(new FileReader("src/test/resources/config/" + configfile));
+	            JSONObject envs = (JSONObject) config.get("environments");
+	            DesiredCapabilities capabilities = new DesiredCapabilities();
+
+	            Map<String, String> envCapabilities = (Map<String, String>) envs.get(browser);
+	            Iterator it = envCapabilities.entrySet().iterator();
+	            while (it.hasNext()) {
+	                Map.Entry pair = (Map.Entry) it.next();
+	                capabilities.setCapability(pair.getKey().toString(), pair.getValue().toString());
+	            }
+
+	            Map<String, String> commonCapabilities = (Map<String, String>) config.get("capabilities");
+	            it = commonCapabilities.entrySet().iterator();
+	            while (it.hasNext()) {
+	                Map.Entry pair = (Map.Entry) it.next();
+	                if (capabilities.getCapability(pair.getKey().toString()) == null) {
+	                    capabilities.setCapability(pair.getKey().toString(), pair.getValue().toString());
+	                }
+	            }
+
+	            String username = System.getenv("BROWSERSTACK_USERNAME");
+	            if (username == null) {
+	                username = (String) config.get("user").toString();
+	            }
+
+	            String accessKey = System.getenv("BROWSERSTACK_ACCESS_KEY");
+	            if (accessKey == null) {
+	                accessKey = (String) config.get("key").toString();
+	            }
+
+	            capabilities.setCapability("browserstack.appiumLogs", "false");
+	            capabilities.setCapability("browserstack.seleniumLogs", "false");
+	            capabilities.setCapability("browserstack.geoLocation", "IN");
+	           
+	            
+	            if (capabilities.getCapability("browserstack.local") != null
+	                    && capabilities.getCapability("browserstack.local") == "true") {
+	                local = new Local();
+	                UUID uuid = UUID.randomUUID();
+	                capabilities.setCapability("browserstack.localIdentifier", uuid.toString());
```

Comment:
Remove this as localIdentifier will get set in the code binding itself on line 102. This is creating conflict and causing errors:
Error: [browserstack.local] is set to true but not started.